templates/Email/MFA/Notification_backupcodeused.ss

```diff
@@ -0,0 +1,10 @@
+<p><%t SilverStripe\\MFA\\Email\\Notification.HELLO 'Hi' %> $Member.Name,</p>
+
+<p>
+	<%t SilverStripe\\MFA\\Email\\Notification.USEDBACKUPCODE 'A recovery code was used to gain access to your account at' %> $AbsoluteBaseURL.<br />
+	<%t SilverStripe\\MFA\\Email\\Notification.CODESLEFT 'You now have {count} codes remaining.' count=$CodesRemaining %>
+</p>
+
+<p>
+    <%t SilverStripe\\MFA\\Email\\Notification.NOTYOU 'If you did not take this action, please contact your system administrator immediately' %>
```

Comment:
Can you ensure each sentence has a full stop at the end?

Reply:
I have done this.